Tasks/AzureStaticWebAppV0/task.json

```diff
@@ -160,6 +160,30 @@
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "Directory location of the Data API source files relative to working directory"
+    },
+    {
+      "name": "azure_static_web_apps_api_token",
```

Comment:
Isn't this already present on line 133? https://github.com/microsoft/azure-pipelines-tasks/pull/20461/files#diff-89f5d6e13c90cb179acb54fc3d877f3852adba33d6761572d45222e3ae4ed91fR133